micronaut-maven-plugin/src/main/resources/dockerfiles/DockerfileNativeStatic

```diff
@@ -22,7 +21,7 @@ COPY classes /home/app/classes
 COPY dependency/* /home/app/libs/
 COPY *.args /home/app/graalvm-native-image.args
 ARG CLASS_NAME
-RUN native-image @/home/app/graalvm-native-image.args --static --libc=musl -H:Class=${CLASS_NAME} -H:Name=application -cp "/home/app/libs/*:/home/app/classes/"
+RUN native-image @/home/app/graalvm-native-image.args --static --target=linux-amd64 --libc=musl -H:Class=${CLASS_NAME} -H:Name=application -cp "/home/app/libs/*:/home/app/classes/"
```

Comment:
Does the Maven plugin allow overriding this?

Reply:
Anything else is not tested/supported. See the rest of the Dockerfile

Reply:
In this regard, I have created #775. Note that they're also X64 only